src/components/WeatherApp.jsx

```diff
@@ -1,111 +1,205 @@
+                  <span>&#8451;</span>
                 </div>
-            </main>
-            <ToastContainer />
+                <p
+                  className={
+                    weatherData.error
+                      ? "tempSub-p-displayErrorSymbol"
+                      : "tempSub-p-removeErrorSymbol"
+                  }
+                >
+                  {isLoading ? (
+                    <BeatLoader color="#252827" size={10} />
+                  ) : weatherData.error ? (
+                    "?"
+                  ) : (
+                    weatherData?.current?.condition?.text
+                  )}
+                </p>
+              </div>
+            </div>
+            <div className="lastSection">
+              <div className="weatherSection">
+                <div>
+                  {isLoading ? (
+                    <BeatLoader color="#252827" size={10} />
+                  ) : weatherData.error ? (
+                    "?"
+                  ) : weatherData?.current?.humidity <= 55 ? (
+                    <FontAwesomeIcon icon={faDroplet} className="dropIcon-1" />
+                  ) : weatherData?.current?.humidity <= 65 &&
+                    weatherData?.current?.humidity > 55 ? (
+                    <FontAwesomeIcon icon={faDroplet} className="dropIcon-2" />
+                  ) : (
+                    <FontAwesomeIcon icon={faDroplet} className="dropIcon-3" />
+                  )}
+                </div>
+                <div>
+                  <h3
+                    className={
+                      weatherData.error
+                        ? "h3-displayErrorSymbol"
+                        : "h3-removeErrorSymbol"
+                    }
+                  >
+                    {isLoading ? (
+                      <BeatLoader color="#252827" size={10} />
+                    ) : weatherData.error ? (
+                      "?"
+                    ) : (
+                      weatherData?.current?.humidity
+                    )}
+                    &nbsp;
+                  </h3>
+                  <span>%</span>
+                  <p>Humidity</p>
+                </div>
+              </div>
+              <div className="weatherSection">
+                <div>
+                  <FontAwesomeIcon icon={faWind} size="2xl" />
+                </div>
+                <div>
+                  <h3
+                    className={
+                      weatherData.error
+                        ? "h3-displayErrorSymbol"
+                        : "h3-removeErrorSymbol"
+                    }
+                  >
+                    {isLoading ? (
+                      <BeatLoader color="#252827" size={10} />
+                    ) : weatherData.error ? (
+                      "?"
+                    ) : (
+                      weatherData?.current?.wind_kph
+                    )}
+                    &nbsp;
+                  </h3>
+                  <span>Km/H</span>
```

Comment:
In future features lets try to have a setting of metric and imperial units. someone might wanna see miles per hour wind.